public/js/question.js

```diff
@@ -1,10 +1,10 @@
-$('#answerform').submit(function(evt) {
+$('#re_answerform').submit(function(evt) {
     evt.preventDefault();
     var ans = $('#answer').val();
     $.ajax({
         type: 'POST',
-        url: '/submitanswer',
-        data: $('#answerform').serialize(),
+        url: '/submitREanswer',
```

Comment:
why changing the route?

Reply:
Need to keep each question type to it's own route, code will become spaghetti if dont do it